src/compas/files/off.py

```diff
@@ -142,7 +112,8 @@ def read(self):
                 continue
 
             if len(parts) == 3:
-                self.v, self.f, self.e = int(parts[0]), int(parts[1]), int(parts[2])
+                self.vertex_count, self.face_count, self.edge_count = \
```

Comment:
in data structures these are called `number_of_...`

Reply:
Might be nice to rename them for consistency. On the other hand, after set they are not used again so maybe remove them all together?